docs/sources/tutorials/quick-start-browser.md

```diff
@@ -206,7 +208,7 @@ import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base';
 import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
 import { initializeGrafanaAgent } from '@grafana/agent-web';
-import { GrafanaAgentTraceExporter } from '@grafana/agent-trace-web';
+import { GrafanaAgentTraceExporter, SessionProcessor } from '@grafana/agent-trace-web';
```

Comment:
Should we name the `SessionProcessor` similarly to `GrafanaAgentTraceExporter` to keep naming consistent?

Reply:
`GrafanaAgentTraceExporter` is named so because it exports *to* the grafana agent, not because it's `GrafanaAgent` related. Buut maybe it wouldn't hurt anyway to rename :thinking: